lib/rsc/story.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import React, { forwardRef } from "react";
+import { useStoryblokState } from "../common/client";
+import { ISbStoryData, StoryblokBridgeConfigV2 } from "../types";
+import { StoryblokComponent } from "./common";
+
+interface StoryblokStoryRSCProps {
```

Comment:
@edodusi Do you think users would need to use this prop on their clients or is just internal?

Reply:
this is just internal, I think no need to export here